src/main/test/com/endava/internship/collections/StudentMapTest.java

```diff
@@ -0,0 +1,683 @@
+package com.endava.internship.collections;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
I don't see dependency in pom for org.assertj

Reply:
Added.